src/components/RecipeEdit.vue

```diff
@@ -528,7 +528,7 @@ const initEmptyRecipe = () => {
     // this.nutrition = {}
     recipe.value = {
         id: 0,
-        name: null,
```

Comment:
@seyfeb this line comes from you. I have not found any side effects but maybe you want to have a look here, though.